devops/kustomize/base/jobs/data-service.yaml

```diff
@@ -0,0 +1,149 @@
+kind: Job
```

Comment:
Provides an easy way to run the Data Service adhoc.  However, you have to manually delete the Job afterwards because Kustomize doesn't support `generateName`.